src/api/helpers/mapCompaniesHouseFields.test.ts

```diff
@@ -0,0 +1,78 @@
+import { mapCompaniesHouseFields, CompanyHouseResponse } from './mapCompaniesHouseFields';
+
+describe('mapCompaniesHouseFields()', () => {
+  // @ts-ignore
```

Comment:
Why do we need this, is it because of `success` and `apiError`? Seems they're not needed?

Reply:
it is because some of the address fields are null and not allowed by the interface (need them as null for testing)

Reply:
done